ast/ast.go

```diff
@@ -2213,13 +2213,15 @@ type ArraySchemaType struct {
 
 // Insert is INSERT statement node.
 //
-//	INSERT INTO {{.TableName | sql}} ({{.Columns | sqlJoin ","}}) {{.Input | sql}}
+//	INSERT INTO {{.TableName | sql}} {{if .InsertOrType}}OR .InsertOrType{{end}} ({{.Columns | sqlJoin ","}}) {{.Input | sql}}
```

Comment:
Is this true?

I guess the following is true:

```suggestion
//	INSERT {{if .InsertOrType}}OR .InsertOrType{{end}}INTO {{.TableName | sql}} ({{.Columns | sqlJoin ","}}) {{.Input | sql}}
```

Reply:
@makenowjust Yes, you're right.